.github/workflows/km-ci-workflow.yaml

```diff
@@ -183,19 +183,32 @@ jobs:
   # parallel with the rest of KM build and test.
   krun-static-build:
     name: Build static KRUN binary
-    runs-on: ${{ fromJSON(needs.start-runners.outputs.run-ons)['ec2'] }}
-    needs: start-runners
+    runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: recursive
+      - name: Set env
```

Comment:
Some comments would be good here. Like we use SHA as a name and fetch the artifact from prior runs if there is one. If not, build afresh and then upload with the SHA name